core/extensions.rs

```diff
@@ -6,11 +6,12 @@ use anyhow::Error;
 use std::borrow::Cow;
 use std::cell::RefCell;
 use std::rc::Rc;
+use std::sync::Arc;
 use std::task::Context;
 use v8::fast_api::FastFunction;
 use v8::ExternalReference;
 
-#[derive(Copy, Clone, Debug)]
+#[derive(Clone, Debug)]
 pub enum ExtensionFileSourceCode {
```

Comment:
I made this `Copy` in #78 just because I could after removing `PathBuf`. It's not important.